### ISSUE TITLE
MXNetError:  Check failed: delay_alloc: 

### ISSUE THREAD

**Issue by blackboxo** [dmlc/dgl#2937]
## 🐛 Bug

![Snipaste_2021-05-24_15-06-53](https://user-images.githubusercontent.com/18532207/119310443-5616f100-bca2-11eb-949d-158b46e26a53.png)

## Environment

 - DGL Version (e.g., 1.0): 0.6.1
 - Backend Library & Version (e.g., PyTorch 0.4.1, MXNet/Gluon 1.3): MXNet 1.7.0.post2
 - OS (e.g., Linux): Linux
 - How you installed DGL (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source): 
 - Python version: 3.6
 - CUDA/cuDNN version (if applicable): 10.2
 - GPU models and configuration (e.g. V100):
 - Any other relevant information:

**Comment by BarclayII** [comment]
Seems that you are running the [SAGEMaker fraud detection example](https://github.com/awslabs/sagemaker-graph-fraud-detection)?  @sojiadeshina

**Comment by blackboxo** [comment]
> Seems that you are running the [SAGEMaker fraud detection example](https://github.com/awslabs/sagemaker-graph-fraud-detection)? @sojiadeshina

yes

**Comment by blackboxo** [comment]
any progress? 😂

**Comment by sojiadeshina** [comment]
Hi, seems you're trying with MXNet 1.7.0 any chance which the original code wasn't tested with. Any chance of using MXNet 1.6.0?

**Comment by blackboxo** [comment]
> Hi, seems you're trying with MXNet 1.7.0 any chance which the original code wasn't tested with. Any chance of using MXNet 1.6.0?

Thanks, but another error happened. "Floating point exception", seems heppened in line 47 " pred = model(node_flow, features[batch_nids.as_in_context(ctx)])" in train_dgl_mxnet_entry_point.py

**Comment by sojiadeshina** [comment]
> > Hi, seems you're trying with MXNet 1.7.0 any chance which the original code wasn't tested with. Any chance of using MXNet 1.6.0?
> 
> Thanks, but another error happened. "Floating point exception", seems heppened in line 47 " pred = model(node_flow, features[batch_nids.as_in_context(ctx)])" in train_dgl_mxnet_entry_point.py

Can you post the full stack trace. I assume you're running this on your own graph

**Comment by blackboxo** [comment]
> > > Hi, seems you're trying with MXNet 1.7.0 any chance which the original code wasn't tested with. Any chance of using MXNet 1.6.0?
> > 
> > 
> > Thanks, but another error happened. "Floating point exception", seems heppened in line 47 " pred = model(node_flow, features[batch_nids.as_in_context(ctx)])" in train_dgl_mxnet_entry_point.py
> 
> Can you post the full stack trace. I assume you're running this on your own graph

Nope, I run on the same IEEE-CIS fraud dataset. It did not print full stach trace, juts aborted and say "Floating point exception".
I think it is a bug in nd.LeakyReLU(h) when size of h is 0
